src/Adapter/Presenter/Order/OrderReturnLazyArray.php

```diff
@@ -48,9 +48,9 @@ class OrderReturnLazyArray extends AbstractLazyArray
 
     /**
      * OrderReturnLazyArray constructor.
-     * @param $prefix
+     * @param string $prefix
      * @param Link $link
-     * @param $orderReturn
+     * @param array $orderReturn
```

Comment:
If you're sure it's everytime an array, you can edit the signature below =)

Reply:
I hesitated but I was not sure, couldn't there be incompatibilities with old versions of PHP? (<7)

Reply:
No, just depends if you pass other thing than an array like `Iterator`, `ArrayAccess`, or a `generator`.
It's maybe a `mixed` instead of an array. Wdyt?

Reply:
you're right we should force an array because OrderReturnPresenter::present checks that the input is an array https://github.com/PrestaShop/PrestaShop/blob/e8079ca22ccb3cb765823081835510ebd7ec9fb4/src/Adapter/Presenter/Order/OrderReturnPresenter.php#L64
I am gonna change it